test/image/render_mocks_cli

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -e
+
+programname=$0
+
+function usage {
+    echo "usage: $programname output_folder [docker:image]"
+    echo "  it renders all the JSON files found in ./test/image/mocks into the output_folder."
+    echo "  if docker:image is provided, rendering is done in the given Docker image."
+    exit 1
+}
+
+# if less than two arguments supplied, display usage
+if [  $# -le 0 ]
+then
+	usage
+fi
+
+for format in png pdf svg eps emf
+do
+  # To obtain reproducible results, we need to use software rendering
+  export LIBGL_ALWAYS_SOFTWARE=true
+  export GALLIUM_DRIVER=softpipe
+
+  # Supply defaults assets to Orca
+  ORCA_OPTS=(--topojson /plotly-geo-assets.js --mathjax /mathjax/MathJax.js)
+
+  # pin Plotly.js' version
+  ORCA_OPTS+=(--plotlyJS "https://cdn.plot.ly/plotly-1.47.2.min.js")
```

Comment:
Hmm. Does the soon-to-be public docker container use `plotly-latest.min.js` by default? Or a tagged version?

Reply:
Yes it will use whatever Orca uses by default which is `plotly-latest.min.js`.

The reason we specify it here is to make sure we use the same version of plotly.js used to generate the baselines.

Reply:
OK, that's great.

Could you add some info under https://github.com/plotly/orca/blob/docker-release/CONTRIBUTING.md#running-tests about the image tests and where to update the plotly.js version used to generate the baselines when needed.

Reply:
Done in f5ce6f9b4231d930c699d635ec38fd8b4785a9f7

Reply:
Fantastic :tada: